tests/functional/sanity_test.py

```diff
@@ -0,0 +1,2 @@
+def test_it():
+    assert True
```

Comment:
This is just the starter functest that the cookiecutter puts in new apps because there has to be at least one test in order to `make functests` to work. We could remove this file and add it to the ignore list, if we don't want this project to have it